DependencyInjection/Configuration.php

```diff
@@ -388,9 +388,33 @@ private function addModelSection(ArrayNodeDefinition $node)
                 ->arrayNode('class')
                     ->addDefaultsIfNotSet()
                     ->children()
-                        ->scalarNode('media')->defaultValue('Application\\Sonata\\MediaBundle\\Entity\\Media')->end()
-                        ->scalarNode('gallery')->defaultValue('Application\\Sonata\\MediaBundle\\Entity\\Gallery')->end()
-                        ->scalarNode('gallery_has_media')->defaultValue('Application\\Sonata\\MediaBundle\\Entity\\GalleryHasMedia')->end()
+                        ->scalarNode('media')
+                            ->defaultValue('Application\\Sonata\\MediaBundle\\Entity\\Media')
+                            ->validate()
+                                ->ifTrue(function ($v) {
```

Comment:
Are you sure the `ifFalse()` method exists? Can't see it in [ExprBuilder](https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/Config/Definition/Builder/ExprBuilder.php).